src/lib.rs

```diff
@@ -54,6 +54,7 @@ pub struct DeviceManagerOptions {
     pub interfaces_directory: String,
     pub store_directory: String,
     pub download_directory: String,
+    pub astarte_ignore_ssl: Option<bool>,
```

Comment:
I add `astarte_ignore_ssl` as an Option because It is a optional key of our `config.toml` file. If it declare only `bool` you should always add the key `astarte_ignore_ssl= false`  in toml otherwise you receive `missing field` error during the parsing of config.

Reply:
Ok, I see!